src/ui/components/controls/pane.vue

```diff
@@ -429,6 +429,8 @@ export default {
             }
         },
         start: function (event) {
+            event.preventDefault(); // stop from firing drag event
```

Comment:
pane isn't draggable, this shouldn't be necessary?

Reply:
Without the preventDefault, it will fire a drag event on the frame, which causes it to stop resizing and turn into a drag frame event. I had to add a preventDefault to the tree and inspector resize handles as well, as it was causing similar issues

Reply:
@dtailor90 that doesn't make sense, can you show me this behavior?  I can't reproduce it locally when I remove this line of code.